sourcegraph/add-sourcegraph-extension.batch.yml

```diff
@@ -0,0 +1,20 @@
+name: add-sourcegraph-extension
+description: Add the Sourcegraph extension as a recommended vscode extension
+on:
+  - repositoriesMatchingQuery: file:.vscode/settings.json
+
+steps:
+  - run: |
+      python /tmp/add-sourcegraph.py ${{repository.search_result_paths}}
+    container: python:3-slim
+    mount:
+      - path: ./add-sourcegraph.py
+        mountpoint: /tmp/add-sourcegraph.py
+# Add your json prettifier of choice here
```

Comment:
```suggestion
```

Reply:
Not a big deal but I don't think this comment applies here.

Reply:
Are you sure? I added it because the script may change the indentation

Reply:
Ha! You're right. 
Ignore this then. Thanks. :)